### ISSUE TITLE
Test loss not improving in most recent PyTorch Lightning version

### ISSUE THREAD

**Issue by dav0dea** [torchmd/torchmd-net#2]
There currently is a problem when using the most recent PyTorch lightning version where the test loss does not increase at all and validation and training loss exhibit spikes during the epochs when we evaluate on the test set.
![image](https://user-images.githubusercontent.com/36135990/115002993-a336c480-9ea5-11eb-8776-48bf69d27731.png)
The spikes in validation and training loss are always exactly the same value and the learning rate during these epochs gets increased to the initial learning rate. PyTorch Lightning likely changed something in the way `trainer.run_test()` works, which we are using as a workaround to allow evaluating the test set during training.

The test set is currently evaluated with:
https://github.com/compsciencelab/torchmd-net2/blob/755336ce23f5342157e753a0e04d744db09269e2/src/module.py#L142-L144

The problem does not occur in PyTorch Lightning version 1.1
